src/picknik_ur_base_config/package.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <package format="3">
   <name>picknik_ur_base_config</name>
-  <version>2.4.0</version>
+  <version>2.5.0</version>
 
-  <description>Base config package for a UR5e that can be simulated with mock hardware</description>
+  <description>Base config package for a UR5e robot</description>
```

Comment:
```suggestion
  <description>Base config package for a UR robot</description>
```

Reply:
Updated here and in lots of other README/package.xmls